libraries/eosiolib/capi/eosio/crypto.h

```diff
@@ -186,6 +199,29 @@ void sha512( const char* data, uint32_t length, struct capi_checksum512* hash );
 __attribute__((eosio_wasm_import))
 void ripemd160( const char* data, uint32_t length, struct capi_checksum160* hash );
 
+/**
+ *  BLAKE2 compression function "F"
+ *  https://eips.ethereum.org/EIPS/eip-152
+ *  returns an output ( BASE^EXP ) % MOD
```

Comment:
Copy paste error?

Reply:
Yeah it is.

Reply:
You mean the mod_exp part right? `returns an output ( BASE^EXP ) % MOD`

Reply:
Yes